static/js/tools/map/fetcher/breadcrumb_denom_stat.ts

```diff
@@ -47,7 +45,7 @@ export function useFetchBreadcrumbDenomStat(
     if (!contextOk) {
       return;
     }
-    if (!ifRatio || breadcrumbDenomStatReady()) {
+    if (breadcrumbDenomStatReady()) {
```

Comment:
probably don't need this anymore either right? Because if any of places or denom sv changes, we should refetch? Same for useFetchDenomStat

Reply:
Same example, if the main stat var change but denom stat does not change, should not re-fetch, so this check is still useful.

Reply:
but the useEffect only triggers for changes to statVar.value.denom? Or will statVar.value.denom update when main stat var is updated?

Reply:
Oh, that's actually right. This check was used when I put the useEffect() in chart_loader. But as it is moved here with the minimal dependencies, it's not needed.

I need to verify this after I merge this back to the full change.